template/index.html

```diff
@@ -3,6 +3,7 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="shortcut icon" href="${require('./src/favicon.ico')}">
```

Comment:
There's a way to make html-loader work just on `<link href="./src/favicon.js">` but I don't know how.